web/app/page.js

```diff
@@ -1,34 +1,39 @@
 "use client";
-import { useState } from "react";
+import { useEffect, useState } from "react";
 import Since from "./components/since";
-import Resetter from "./components/resetter";
 
 let allergenNames = [
-  { name: "Milk", time: new Date() },
-  { name: "Eggs", time: new Date() },
-  { name: "Nuts", time: new Date() },
-  { name: "Fish", time: new Date() },
-  { name: "Wheat", time: new Date() },
-  { name: "Soy", time: new Date() },
-  { name: "Sesame", time: new Date() },
+  { name: "Milk" },
+  { name: "Eggs" },
+  { name: "Nuts" },
+  { name: "Fish" },
+  { name: "Wheat" },
+  { name: "Soy" },
+  { name: "Sesame" },
```

Comment:
```suggestion
  "Milk",
  "Eggs",
  "Nuts",
  "Fish",
  "Wheat",
  "Soy",
  "Sesame",
```

this simplifies things a bit since we only have a single value for each of these